client/my-sites/site-settings/settings-newsletter/main.tsx

```diff
@@ -44,220 +44,172 @@ type Fields = {
 				/>
-			</Card>
-			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
-			<SettingsSectionHeader
-				disabled={ disabled }
-				id="email-settings"
-				isSaving={ isSavingSettings }
-				onButtonClick={ handleSubmitForm }
-				showButton
-				title={ translate( 'Email' ) }
-			/>
-			<Card className="site-settings__card">
-				<FeaturedImageEmailSetting
+				<NewsletterCategoriesSection
 					disabled={ disabled }
+					newsletterCategoryIds={
+						settings?.wpcom_newsletter_categories || defaultNewsletterCategoryIds
+					}
+					newsletterCategoriesEnabled={ settings?.wpcom_newsletter_categories_enabled }
 					handleToggle={ handleToggle }
-					value={ wpcom_featured_image_in_email }
+					updateFields={ updateSettings }
 				/>
-			</Card>
-			<Card className="site-settings__card">
-				<ExcerptSetting
-					disabled={ disabled }
-					updateFields={ updateFields }
-					value={ wpcom_subscription_emails_use_excerpt }
-				/>
-			</Card>
-			<Card className="site-settings__card">
-				<ReplyToSetting
-					disabled={ disabled }
-					updateFields={ updateFields }
-					value={ jetpack_subscriptions_reply_to }
-				/>
-			</Card>
-
-			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
-			<SettingsSectionHeader
-				id="newsletter-categories-settings"
-				title={ translate( 'Newsletter categories' ) }
-				showButton
-				onButtonClick={ handleSubmitForm }
-				disabled={ disabled }
-				isSaving={ isSavingSettings }
-			/>
-			<NewsletterCategoriesSection
-				disabled={ disabled }
-				newsletterCategoryIds={ fields.wpcom_newsletter_categories || defaultNewsletterCategoryIds }
-				newsletterCategoriesEnabled={ fields.wpcom_newsletter_categories_enabled }
-				handleToggle={ handleToggle }
-				updateFields={ updateFields }
-			/>
 
-			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
-			<SettingsSectionHeader
-				disabled={ disabled }
-				id="messages"
-				isSaving={ isSavingSettings }
-				onButtonClick={ handleSubmitForm }
-				showButton
-				title={ translate( 'Messages' ) }
-			/>
-			<Card className="site-settings__card">
-				<EmailsTextSetting
+				{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
+				<SettingsSectionHeader
 					disabled={ disabled }
-					updateFields={ updateFields }
-					value={ subscription_options }
+					id="messages"
+					isSaving={ isSavingSettings }
+					onButtonClick={ handleSubmitForm }
+					showButton
+					title={ translate( 'Messages' ) }
 				/>
-			</Card>
-		</form>
-	);
-} );
+				<Card className="site-settings__card">
+					<EmailsTextSetting
+						disabled={ disabled }
+						updateFields={ updateSettings }
+						value={ settings?.subscription_options }
+					/>
+				</Card>
+			</form>
+		);
+	}
+);
 
 const NewsletterSettings = () => {
 	const translate = useTranslate();
-
+	const siteId = useSelector( getSelectedSiteId ) ?? 0;
```

Comment:
Why is it necessary to fall back to `0`? Couldn't this bleed somewhere unexpected and cause issues?

Reply:
It is not I was just running into issues with typescript when I used `const siteId = useSelector( getSelectedSiteId )`